boards/esp32-ethernet-kit-v1_0/doc.txt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ * Copyright (C) 2020 Google LLC
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_esp32_esp-ethernet-kit-v1_0 v1.0 Board
+ * @ingroup     boards_esp32_esp-ethernet-kit
+ * @brief       Support for for Espressif ESP32-Ethernet-Kit v1.0
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ * @author      Erik Ekman <eekman@google.com>
+
+## <a name="toc"> Table of Contents </a>
+
+1. [Overview](#overview)
+2. [Hardware](#hardware)
+    1. [Board Configuration](#board_configuration)
+    2. [Board Pinout](#pinout)
+3. [Other Documentation Resources](#other-resources)
+
+## <a name="overview"> Overview </a> &nbsp;&nbsp; [[TOC](#toc)]
+
+The Espressif [ESP32-Ethernet-Kit](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/hw-reference/esp32/get-started-ethernet-kit-v1.0.html) is a development board that uses the ESP32-WROVER-B module. Most important features of the board are
+
+- 100 Mbps Ethernet via IP101G PHY
+- USB bridge with JTAG interface
+
+Furthermore, some GPIOs are broken out for extension. The USB bridge based on FDI FT2232HL provides a JTAG interface for OCD debugging through the USB interface.
+
```

Comment:
A link to the information on how to flash and debug might be helpful here when crawling through the documentation.
```suggestion
For flashing and debugging the board, see \ref boards_esp32_esp-ethernet-kit common board documentation.

```